src/main/java/com/aws/greengrass/deployment/activator/DeploymentActivator.java

```diff
@@ -31,10 +34,12 @@ public abstract class DeploymentActivator {
     protected final Kernel kernel;
     protected final DeploymentDirectoryManager deploymentDirectoryManager;
     protected static final Logger logger = LogManager.getLogger(DeploymentActivator.class);
+    protected final ComponentStore componentStore;
 
     protected DeploymentActivator(Kernel kernel) {
         this.kernel = kernel;
         this.deploymentDirectoryManager = kernel.getContext().get(DeploymentDirectoryManager.class);
+        this.componentStore = kernel.getContext().get(ComponentStore.class);
```

Comment:
unused?

Reply:
It's only used in `DefaultActivator`. Updated